### PR TITLE
Add new query 'claims' + snapshot tests

### PR DESCRIPTION
**Pull request by anthansson**
# Jira Issue: [MX-182]

## What?
- New query 'claims' added that lets clients fetch all their claims. Initially had this [old branch](https://github.com/HedvigInsurance/giraffe/pull/1276) but created this fresh one as there was too many conflicts as we have swapped GraphQL gen library. 

## Why?
- Ultimately will allow members to follow the progress of their claims

## Optional checklist
- [ ] Unit tests written
- [x] Tested locally
- [x] Codescouted

[MX-182]: https://hedvig.atlassian.net/browse/MX-182

### CODE REVIEW

File: src/resolvers/claims.test.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+describe('Query.claims', () => {
+
+  it('works for zero claims', async () => {
+    upstream.claimService.getMemberClaims = () => Promise.resolve([])
+    const claims = await QUERIES.claims()
+    expect(claims).toMatchSnapshot()
+  })
+
+  it('works for claim with contract with agreement', async () => {
+    //context.upstream.claimService.createClaim = () => Promise.resolve({claimId: MEMBER_ID})
```
Comment (hedvigoscar) [inline src/resolvers/claims.test.ts:18]:
```suggestion
```

File: src/schema.graphqls [hunk 2]
```diff
@@ -1019,3 +1022,30 @@ type Address {
...
+  id: String!
+  contract: Contract
+  status: ClaimStatus!
+  outcome: ClaimOutcome
+  submittedAt: Instant!
+  closedAt: Instant
+  files: [File!]!
+  signedAudioURL: String
+  payout: MonetaryAmountV2
+  _contractId: ID
```
Comment (hedvigoscar) [inline src/schema.graphqls:1037]: This is an implementation detail of `giraffe` that we should not leak 🙂 
```suggestion
```

**Review by hedvigoscar — APPROVED** [review]
Looks mostly sensible to me. Check the comments, after they are addressed you can merge 🙂

File: src/schema.graphqls (hunk 2 above)
Comment (anthansson) [reply src/schema.graphqls:1037]: We put it there when you helped me with this.If I remove it I need to do some work arounds?

Comment (anthansson) [reply src/schema.graphqls:1037]: removed 🦖

**Pull request merged**